inference-engine/tests/ngraph_functions/src/scatter_ND_update.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<ngraph::Node> makeScatterNDUpdate(const ngraph::Output<Node> &in,
+                                                  const element::Type& indicesType,
+                                                  const std::vector<size_t>& indicesShape,
+                                                  const std::vector<size_t>& indices,
+                                                  const ngraph::Output<Node> &update) {
+    auto indicesNode = std::make_shared<ngraph::opset1::Constant>(indicesType, indicesShape, indices);
```

Comment:
Why did you add this method? It doesn't work.

Reply:
Thank you for pointing out. I am implementing ScatterUpdate layers together. When I run single layer test, I showed ScatterNDUpdate and ScatterElementsUpdate ref are not support yet in ngraph. so I [commented out the test](https://github.com/openvinotoolkit/openvino/blob/e891d94ba09adb93bc8dbfadd59f51db8d6047ea/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/scatter_ND_update.cpp#L135). and will open when support is available in ngraph.